mobile/app/src/main/java/com/mylook/mylook/explore/LocationValidator.java

```diff
@@ -0,0 +1,27 @@
+package com.mylook.mylook.explore;
+
+import android.location.Location;
+import android.util.Log;
+
+import com.mylook.mylook.entities.Article;
+
+public class LocationValidator {
+
+    private static final double NEARBY_RADIUS = 5d;
```

Comment:
radio de 5 metros seria esto?

Reply:
5km, con eso puedo encontrar algunas tiendas
obviamente eso hay que cambiarlo despues

Reply:
o parametrizarlo, pero viaje

Reply:
E/Article: checkIfNearby: -31.4105377 - -64.1940184
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.mylook.mylook, PID: 6756
    java.lang.NullPointerException: Attempt to invoke virtual method 'double android.location.Location.getLatitude()' on a null object reference
        at com.mylook.mylook.explore.LocationValidator.checkIfNearby(LocationValidator.java:24)
        at com.mylook.mylook.explore.ExploreService.createExploreArticleList(ExploreService.java:71)
        at com.mylook.mylook.explore.ExploreFragment.lambda$getArticles$5$ExploreFragment(ExploreFragment.java:309)
        at com.mylook.mylook.explore.-$$Lambda$ExploreFragment$UBI3hGRzNE28FRpJNoKBybcSHhk.onComplete(Unknown Source:6)
        at com.google.android.gms.tasks.zzj.run(Unknown Source:4)
        at android.os.Handler.handleCallback(Handler.java:869)
        at android.os.Handler.dispatchMessage(Handler.java:101)
        at android.os.Looper.loop(Looper.java:206)
        at android.app.ActivityThread.main(ActivityThread.java:6749)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:845)

llego a pedirme permisos y exploto
probe segunda vez (ya tenia dados los permisos) y prendi la ubicacion (sin que me pidiera) pero tambien crasheo

Reply:
Pasa que depende, en el centro 5 Km es una banda. En mendiolaza tenes dos panaderías a esa distancia

Reply:
Pero lo dejaría asi